recipes/Gdal_CSharp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.0" %}
+{% set gdalversion = "3.2.2" %}
```

Comment:
Should these be the same version number? I don't see a reason for the `1.0.0` but I might be missing something.

Reply:
This was a request from the GDAL team - 3.2.2 is the version number of the GDAL source that is used for the build. They want the version number of the gdal-csharp package to be different from the source version. It allows flexibility and the ability to have out-of-synch bindings releases in the future

Reply:
This is rather odd as there is no place where we have the mapping of 1.0.0 = 3.2.2 recorded. If the GDAL team wants a separate release cycle, then they need to release the code separately, right?

Reply:
Also,. the conda-forge bots won't be able to update this feedstock as versions change.

Reply:
We will live with manual update tasks for the time being. The long range plan - once this is bedded in - is to separate the source of the bindings into a new repository to make it more maintainable.